localization/pose_initializer/src/pose_initializer_core.cpp

```diff
@@ -46,6 +46,34 @@ PoseInitializer::PoseInitializer()
 {
   enable_gnss_callback_ = this->declare_parameter("enable_gnss_callback", true);
 
+  this->declare_parameter("initialpose_particle_covariance");
+  std::vector<double> initialpose_particle_covariance =
+    this->get_parameter("initialpose_particle_covariance").as_double_array();
+  for (std::size_t i = 0; i < initialpose_particle_covariance.size(); ++i) {
+    initialpose_particle_covariance_[i] = initialpose_particle_covariance[i];
+  }
```

Comment:
```suggestion
  initialpose_particle_covariance_ =
    this->declare_parameter<std::vector<double>>("initialpose_particle_covariance");
```
 if `initialpose_particle_covariance_` is `std::vector<double>`

Reply:
@wep21 Thank you for your suggestion.
`initialpose_particle_covariance_` is `std::arrary<double, 36>`, so it could not be applied as is.
But thanks to you, I was able to rewrite it a little simpler.　https://github.com/autowarefoundation/autoware.universe/pull/279/commits/2ff2ea554c13cc687eb9e4dc7ad500fd2ad0eb5f